src/steps/resource-manager/batch/index.ts

```diff
@@ -120,28 +122,31 @@ export async function fetchBatchApplications(
     async (batchAccountEntity) => {
       const { id, name } = batchAccountEntity;
       const resourceGroup = resourceGroupName(id, true)!;
-
-      await client.iterateBatchApplications(
-        {
-          resourceGroupName: resourceGroup,
-          batchAccountName: name as string,
-        },
-        async (batchApplication) => {
-          const batchApplicationEntity = createBatchApplicationEntity(
-            webLinker,
-            batchApplication,
-          );
-          await jobState.addEntity(batchApplicationEntity);
-
-          await jobState.addRelationship(
-            createDirectRelationship({
-              _class: RelationshipClass.HAS,
-              from: batchAccountEntity,
-              to: batchApplicationEntity,
-            }),
-          );
-        },
-      );
+      const rawData = getRawData<BatchAccount>(batchAccountEntity);
```

Comment:
nit: use `batchAccountEntity` and `batchAccount` as variable names, as described in the **Fetch Child Entities** step pattern documentation https://github.com/JupiterOne/sdk/blob/main/docs/integrations/step-patterns.md#fetch-child-entities-step-pattern

```suggestion
      const batchAccount = getRawData<BatchAccount>(batchAccountEntity);
```